lecture/models.py

```
@@ -1,26 +1,57 @@
-        if not self.slug and self.title:
-            self.slug = slugify(self.title)  # Auto-generate slug from title
+        if not self.slug:
+            self.slug = slugify(self.title)
         super().save(*args, **kwargs)
+
+
+class LecturePDF(BaseModel):
+    lecture = models.ForeignKey(
+        Lecture,
+        on_delete=models.CASCADE,
+        related_name='pdf_files'
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)